src/data/monsters/wolf/index.js

```diff
@@ -1,4 +1,5 @@
 import WolfSprite from './wolf.png';
+import WolfFloppedSprite from './wolf-flopped.png';
```

Comment:
Is `Flopped` a typo or intended? Not sure while reading this.

Reply:
https://imagemagick.org/script/command-line-options.php#flop